drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -177,12 +184,12 @@ function DragNDropTemplates(url_name) {
         // and render its placed items as children
         var item_wrapper = 'div.item-wrapper';
         var items_in_zone = [];
+        var is_item_in_zone = function(i) { return i.is_placed && (i.zone === zone.uid); };
         if (zone.align !== 'none') {
             item_wrapper += '.item-align.item-align-' + zone.align;
-            var is_item_in_zone = function(i) { return i.is_placed && (i.zone === zone.uid); };
             items_in_zone = $.grep(ctx.items, is_item_in_zone);
         }
-
+        var items_in_zone_descriptions = $.grep(ctx.items, is_item_in_zone);
```

Comment:
@dovinmu The same array is calculated two lines above. Can you change the code to avoid calculating it twice?